src/zope/testrunner/tests/testrunner-subunit.rst

```diff
@@ -0,0 +1,721 @@
+Subunit Output
+==============
+
+Subunit is a streaming protocol for interchanging test results. More
+information can be found at https://launchpad.net/subunit.  To enable
+support for ``subunit``, install it and its dependencies via the
+``zope.testrunner`` extra:
+
+    $ pip install zope.testrunner[subunit]
+
+or directly:
+
+    $ pip install python-subunit
+
+First we need to make a temporary copy of the entire testing directory:
+
+    >>> import os.path, sys, tempfile, shutil
+    >>> tmpdir = tempfile.mkdtemp()
```

Comment:
Please `prefix='test-zope.testrunner-'`.

(I'm probably wildly inconsistent with my suggested prefixes.  The main thing I want is that it mention zope.testrunner, and that it was created from test code, to help narrow down the source of mysterious leftover temp files.)

Reply:
(The experience of running a Zope 3 buildbot on a machine with a small-by-modern-standards /tmp partition left scars.)